wiring_globals/src/wiring_globals_i2c.cpp

```diff
@@ -8,16 +8,52 @@
 
 #ifndef SPARK_WIRING_NO_I2C
 
+namespace {
+
+HAL_I2C_Config defaultWireConfig() {
+	HAL_I2C_Config config = {
+		.size = sizeof(HAL_I2C_Config),
+		.version = HAL_I2C_CONFIG_VERSION_1,
+		.rx_buffer = new (std::nothrow) uint8_t[I2C_BUFFER_LENGTH],
```

Comment:
This might be `nullptr`.

Reply:
https://github.com/particle-iot/device-os/pull/2022#discussion_r381793729